### DISCUSSION TITLE
Account Aggregator flow through APIs

### DISCUSSION THREAD

**Discussion by vishalsingh020997** [ONDC-Official/ONDC-FIS-Specifications#76] [Q&A]
Hi, I was not able to find AA flow in the draft-loan APIs. We are sending the AA id in the form, but couldn't see lender app sending the consent handle or buyer app informing about consent to lender app.

**Comment by Antriksh-Parmar** [comment]
@vishalsingh020997 AA integration is off ONDC network.

**Reply by vishalsingh020997** [reply]
@Antriksh-Parmar So you mean we have to integrate with each and every lender separately off the network?

**Reply by jainish-shah-2212** [reply]
@Antriksh-Parmar  can you respond here ? We are getting this same issue , need a better understanding for this. Also I was going over Figma link for sample App , there I see that each BPP will send a list of AA's that it supports , according to my understanding this should Ideally be part of on_search API , can you elaborate if my understanding is correct ?
